packages/vscode-graphql/esbuild.js

```diff
@@ -11,10 +11,46 @@ build({
   minify: arg === '--minify',
   platform: 'node',
   outdir: 'out/',
-  external: ['vscode'],
   format: 'cjs',
   sourcemap: true,
   watch: isWatchMode,
+  // Avoid bundling @vue/compiler-sfc's dynamic dependencies
```

Comment:
ah, is there any way we can more selectively import it by chance?

this only removes it for `vscode-graphql` users, and we have vim, emacs, sublime users as well, who may end up with a bloated bundle over this. are they dynamically loaded into the runtime as needed at least?

Reply:
My comment was a bit misleading. I specified these packages as `external` not to reduce bundle size, but to avoid bundling errors. They are template preprocessors and `require()`d at runtime by `@vue/consolidate`. `@vue/consolidate` does not specify the preprocessors as `dependencies` in its package.json, so users need to install them separately.

However, in this case, esbuild tries to load the preprocessors and fails with `package not found` error. 

If the users just install and use `graphql-language-service-cli`, no extra packages installed.